packages/@sanity/desk-tool/src/components/paneItem/PaneItem.tsx

```diff
@@ -21,6 +22,18 @@ interface PaneItemProps {
   schemaType?: SchemaType
 }
 
+const PaneItemCard = styled(Card)`
+  /* TextWithTone uses its own logic to set color, and we therefore need 
+  to override this logic in order to set the correct color in different states */
+  &[data-selected],
+  &[data-pressed],
+  &:active {
+    [data-ui='TextWithTone'] {
```

Comment:
Any reason to prefer this over importing `TextWithTone` and using `${TextWithTone}` instead? That would make the dependency more explicit and easier to maintain, but might be benefits with this I'm not aware of.